src/theory/arith/nl/transcendental/proof_checker.h

```diff
@@ -49,11 +49,23 @@ class TranscendentalProofRuleChecker : public ProofRuleChecker
                      const std::vector<Node>& args) override;
   /**
    * Helper method to construct (t >= lb) AND (t <= up)
+   *
+   * @param t term in the above formula
+   * @param lb term in the above formula
+   * @param ub term in the above formula
+   * @return the formula of the form above
    */
   Node mkBounds(TNode t, TNode lb, TNode ub);
   /**
    * Helper method to construct a secant plane:
    * evall + ((evall - evalu) / (l - u)) * (t - l)
+   *
+   * @param t term in the above formula
+   * @param l term in the above formula
+   * @param u term in the above formula
+   * @param evall term in the above formula
+   * @param evalu term in the above formula
+   * @return the formula of the form above
```

Comment:
This is not very helpful. Is the secant plane constructed for `t`? What are the other nodes?